exspy/doc/intro.rst

```diff
@@ -0,0 +1,104 @@
+
+What is Spectrospy
```

Comment:
Do you want to rename `Spectrospy` to `exSpy` now or leave to when the repository is setup?